packages/apps/graph-client/src/constants/search.ts

```diff
@@ -0,0 +1,38 @@
+export enum SearchType {
+  GasEstimation = 'gasEstimation',
+  Account = 'account',
+  Block = 'block',
+  Transactions = 'request-key',
+  Event = 'event',
+}
+
+export const searchTypeLabels: Record<string, string> = {
+  'request-key': 'Request Key',
+  account: 'Account',
+  event: 'Event Name',
+  block: 'Block Hash',
+  gasEstimation: 'Cmd',
```

Comment:
`Command` for consistency (we try to write full names/properties everywhere)

Reply:
Fixed